server/src/main/java/keywhiz/service/daos/SecretDAO.java

```diff
@@ -395,6 +395,28 @@ public Optional<SecretSeriesAndContent> getSecretByName(DSLContext dslContext, S
     return Optional.empty();
   }
 
+  /**
+   * @param name of secret series to look up secrets by.
```

Comment:
nit: missing param version (it's also not `checkArgument`'d below)

Reply:
Added! I went to add a `checkArgument` but I wasn't sure what might constitute an invalid/empty `version`, since it's just a Long.